README.md

```diff
@@ -229,6 +239,25 @@ the JWTs in a different application.
 You can see the details in
 [`packages/shopping/src/application.ts`](./packages/shopping/src/application.ts).
 
+### Reset Password
+
+This repository includes a forgot password and reset password functionality that
+illustrates how shoppers can reset their password in the case they forgot them.
+Shoppers can either reset their password while logged in or locked out of the
+application. For this functionality we use Nodemailer. Please see
+https://nodemailer.com/usage/using-gmail/ if you're planning to use Nodemailer
+with GMAIL. Additionally, to manage environment variables we use `dotenv`,
+therefore, you must create a `.env` file in the root of the project with the
+below contents:
+
+```dotenv
+SMTP_PORT=587
+SMTP_SERVER=smtp.gmail.com
+APPLICATION_URL=http://localhost:3000 <endpoint-to-the-page-with-reset-password-form>
```

Comment:
`APPLICATION_URL=http://localhost:3000/<endpoint-to-the-page-with-reset-password-form>`